tests/test_authentication.py

```diff
@@ -270,7 +270,7 @@ def test_extended_model_claim_mapping(self):
             self.assertEqual(user.first_name, "John")
             self.assertEqual(user.last_name, "Doe")
             self.assertEqual(user.email, "john.doe@example.com")
-            self.assertEqual(user.profile.employee_id, 182)
+            self.assertEqual(user.employee_id, 182)
```

Comment:
This seems to be in the wrong direction. The assertion on `user.profile.employee_id` should work.